### PR TITLE
Pychop fixes

### PR DESCRIPTION
**Pull request by AndreiSavici**
**Description of work.**

Minor fixes for PyChop, to match the documentation page

**To test:**
Try the command line from the updated version of https://docs.mantidproject.org/nightly/interfaces/direct/PyChop.html

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
It works as advertised, but there are some minor comments.

File: qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/PyChopGui.py [hunk 1]
```diff
@@ -721,7 +721,7 @@ def showText(self):
         self.txtloop = QEventLoop()
         self.txtloop.exec_()
 
-    def saveText(self):
+    def saveText(self, *args):
```
[the remark this answers is not in the document]
Comment (AndreiSavici) [reply qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/PyChopGui.py:725]: https://doc.qt.io/qt-5/qabstractbutton.html#clicked Button clicked now returns a flag for "checked" state. Need to have the correct number of parameters

File: qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/PyChopGui.py [hunk 2]
```diff
@@ -694,7 +694,7 @@ def genText(self):
                     txt += self._gen_text_ei(ei, obj)
         return txt
 
-    def showText(self):
+    def showText(self, *args):
```
[the remark this answers is not in the document]
Comment (AndreiSavici) [reply qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/PyChopGui.py:697]: See below

File: qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/Instruments.py [hunk 3]
```diff
@@ -899,17 +898,45 @@ def calculate(cls, *args, **kwargs):
...
+        ! resolution function. By passing etrans='polynomial', the calculator estimates the
+        ! resolution for etrans=np.arange(-Ei, Ei, Ei*0.01) then fits it to a cubic polynomial.
+        ! The resolution is then an array with coefficients, from the lowest power.
+        !
+        ! res, flux = PyChop2.calculate(inst='cncs', variant='High flux', freq=240, ei=1.5, etrans='polynomial')
+        !
         ! The results are returned as tuple: (resolution, flux)
         """
         argdict = argparser(args, kwargs, ['inst', 'package', 'frequency', 'ei', 'etrans', 'variant'])
+        print(argdict)
```
[the remark this answers is not in the document]
Comment (AndreiSavici) [reply qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/Instruments.py]: Removed

**Review by peterfpeterson — APPROVED** [review]

**Pull request merged**